src/website/shared/listeners/cache_suggestions.py

```diff
@@ -179,6 +180,24 @@ def is_version_affected(version_statuses: list[str]) -> Version.Status:
     return result
 
 
+def get_src_position(derivation: NixDerivation) -> str | None:
+    """
+    Get the GitHub URL pointing to the exact source file used for the evaluation of this derivation.
+    E.g. https://github.com/NixOS/nixpkgs/blob/0e8be3827d0298743ba71b91eea652d43d7dc03d/pkgs/by-name/he/hello/package.nix#L47
+    """
+    if derivation.metadata and derivation.metadata.position:
+        rev = urllib.parse.quote(derivation.parent_evaluation.commit_sha1)
+        # position is something like `/tmp/tmpfh7ff2xs/pkgs/development/python-modules/qemu/default.nix:67`
+        position_match = re.match(
+            r"/tmp/[^/]+/(.+):(\d+)", derivation.metadata.position
```

Comment:
This won't scale with https://github.com/Nix-Security-WG/nix-security-tracker/pull/451, because where we evaluate is essentially a runtime configuration, but which may change between deployments... :/

@raitobezarius maybe we can scrape that out already at evaluation time?

Reply:
Oh I understand. Makes the distinction pretty much impossible then, right? I'll add a FIXME for now.

Reply:
If the eval store is in memory, I don't see why we cannot do that computation, it's just a string clean up?

Reply:
We would need to know the location then. Is the config value accessible in cache_suggestions? Also this would mean that all evaluations get invalidated, once you change the location of the eval store.

Reply:
The config value is accessible but in case of config change will get invalidated. This is why we should do it earlier where we know the value at the time. But also I just realised this is not necessary config store but location of the Nixpkgs checkout, so this can definitely just stripped off during eval-ingestion. We'll leave it at that now because it won't help for the demo.